src/test/java/org/cyclonedx/BomXmlGeneratorTest.java

```diff
@@ -89,6 +89,17 @@ public void schema11WithDependencyGraphGenerationTest() throws Exception {
         Assert.assertTrue(parser.isValid(file, CycloneDxSchema.Version.VERSION_11));
     }
 
+    @Test
+    public void schema11WithVulnerabilityTest() throws Exception {
```

Comment:
Q: should we have a test for schema12 here as well? (with the same vuln xml)

Reply:
> Q: should we have a test for schema12 here as well? (with the same vuln xml)

Makes sense to me. Added: b020ca1